src/main/java/com/epam/tc/hw4/pages/ElementsPage.java

```diff
@@ -0,0 +1,55 @@
+package com.epam.tc.hw4.pages;
+
+import java.util.List;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+
+
+public class ElementsPage extends BasePage {
+
+    protected WebDriver driver;
```

Comment:
You don't need this line, since you've already have driver initialization in constructor

Reply:
I fixed